src/viking.js

```diff
@@ -1,11 +1,101 @@
 // Soldier
-function Soldier() {}
+function Soldier(healthArg, strengthArg) {
+  this.health = healthArg;
+  this.strength = strengthArg;
+}
+Soldier.prototype.attack = function (){
+  return this.strength;
+ };
+Soldier.prototype.receiveDamage = function(damageArg){
+  this.health -=  damageArg;
+}
 
 // Viking
-function Viking() {}
+Viking.prototype = Object.create(Soldier.prototype); 
```

Comment:
This line of code would go after you have created the function constructor `function Viking..`
And there is one line missing `Viking.prototype.constructor = Viking;`